crowdloan/kusama.json

```diff
@@ -181,8 +181,8 @@
     "icon": "https://raw.githubusercontent.com/polkadot-js/apps/master/packages/apps-config/src/ui/logos/nodes/integritee.svg",
     "rewardRate": 10
   },
-  {
-    "paraid": "2100",
+   {
```

Comment:
Could you please remove redundant spacing so that we can keep JSON formatted & clean?

Reply:
Done